exotations/task_maps/task_map/CMakeLists.txt

```diff
@@ -53,13 +53,14 @@ target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
 add_dependencies(${PROJECT_NAME} ${PROJECT_NAME}_initializers)
 
 pybind_add_module(${PROJECT_NAME}_py MODULE src/task_map_py.cpp ${SOURCES})
-install(TARGETS ${PROJECT_NAME}_py LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION})
+install(TARGETS ${PROJECT_NAME}_py LIBRARY DESTINATION ${CATKIN_GLOBAL_PYTHON_DESTINATION})
```

Comment:
We have to. The ```*.so``` files have to in python path for them to be recognized as modules. The package destination is a subfolder of the global destination. All the python files go there and it requires a ```_init__.py``` to be recognized as a module.